src/IOCapture.jl

```diff
@@ -3,10 +3,12 @@ using Logging
 import Random
 
 """
-    IOCapture.capture(f; rethrow=Any, color=false)
+    IOCapture.capture(f; rethrow=Any, color=false, pass_through=false)
```

Comment:
Pure bikeshed: maybe `passthrough`? I also found an example of what I think is the same kind of API in [IPython's API](https://ipython.readthedocs.io/en/stable/api/generated/IPython.utils.capture.html#IPython.utils.capture.capture_output), where I presume the `display` argument maybe does what this does. So two more options, but I don't have a strong opinion. @goerz do you want to stick with `pass_through` or change to one of those?

Reply:
I have no preference on the name, either. Let's do `passthrough`